move-vm-support/src/ss58_address.rs

```diff
@@ -0,0 +1,114 @@
+//! SS58 address format converter for Substrate and Move accounts.
+//! Based on the Pontem solution.
```

Comment:
We can give some credit in the final Readme.
```suggestion
```

Reply:
Changed.